src/test/java/com/adobe/epubcheck/ocf/OCFCheckerTest.java

```diff
@@ -384,4 +398,27 @@ public void testInvalidLoremSCH30()
 
     assertTrue(testReport.hasInfoMessage(VERSION_STRING));
   }
+
+  @Test
+  public void testInvalidCompressionMethod()
+  {
+    ValidationReport testReport = testOcfPackage("/30/expanded/invalid/ocf-compression/",
+        EPUBVersion.VERSION_3);
+
+    if (2 != testReport.getErrorCount() || 0 != testReport.getWarningCount())
+    {
+      outWriter.println(testReport);
+    }
+    if (testReport.errorList.size() > 0)
+    {
+      assertTrue(testReport.errorList.get(0).message
+          .contains("value of attribute \"Method\" is invalid; must be equal to \"0\" or \"8\""));
+      assertTrue(testReport.errorList.get(1).message
```

Comment:
this can possibly throw an NPE since we didn't assert at this point that there are at least two errors…

Reply:
>In fact, I don't think EPUBCheck does use a catalog resolver! As far as I know this catalog file is mostly here to be used by third-party tools, e.g. to configure XML editors.

Then, does epubcheck always retreive schemas?